connector_x/include/Commands.h

```diff
@@ -31,7 +31,11 @@ enum class CommandType
     // W
     RadioSend = 12,
     // R
-    RadioGetLatestReceived = 13
+    RadioGetLatestReceived = 13,
+    // R
+    GetColor = 14,
+    // R
+    GetPort = 15
```

Comment:
Need to create new, empty `CommandGetColor` and `CommandGetPort` structs

Reply:
Also need to add handlers to CommandParser.h

Reply:
> Also need to add handlers to CommandParser.h

Resolved conversation for original comment, unsure how to do the second comment

Reply:
Just add new cases within CommandParser.h > `parseCommand` to handle setting the two command structs. Don't forget to add them to the commandData union too.

Reply:
> Just add new cases within CommandParser.h > `parseCommand` to handle setting the two command structs.

Still need this